openquake/hazardlib/tests/gsim/mgmpe/m9_basin_term_test.py

```diff
@@ -0,0 +1,110 @@
+# The Hazard Library
+# Copyright (C) 2012-2023 GEM Foundation
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import numpy as np
+import unittest
+
+from openquake.hazardlib.tests.gsim.mgmpe.dummy import new_ctx
+from openquake.hazardlib.contexts import simple_cmaker
+from openquake.hazardlib import valid
+from openquake.hazardlib.imt import PGA, PGV, SA
+from openquake.hazardlib.const import TRT, IMC
+from openquake.hazardlib.gsim.mgmpe.m9_basin_term import M9BasinTerm
+
+ae = np.testing.assert_equal
+aae = np.testing.assert_almost_equal
+
+
+class M9BasinTermTestCase(unittest.TestCase):
+
+    def test_instantiation(self):
+        mgmpe = M9BasinTerm(gmpe_name='KuehnEtAl2020SInter')
+
+        # Check the assigned IMTs
+        expected = set([PGA, SA, PGV])
+        self.assertTrue(mgmpe.DEFINED_FOR_INTENSITY_MEASURE_TYPES == expected,
+                        msg='The assigned IMTs are incorrect')
+        # Check the TR
+        expected = TRT.SUBDUCTION_INTERFACE
+        self.assertTrue(mgmpe.DEFINED_FOR_TECTONIC_REGION_TYPE == expected,
+                        msg='The assigned TRT is incorrect')
+        # Check the IM component
+        expected = IMC.RotD50
+        self.assertTrue(mgmpe.DEFINED_FOR_INTENSITY_MEASURE_COMPONENT ==
+                        expected, msg='The IM component is wrong')
+        # Check the required distances
+        expected = {'rrup'}
+        self.assertTrue(mgmpe.REQUIRES_DISTANCES == expected,
+                        msg='The assigned distance types are wrong')
```

Comment:
`assertTrue` gives a much worse error message than `np.testing.assert_equal` and should never be used

Reply:
Using np.testing.assert.equal now instead